packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -4579,7 +4579,10 @@ addEventListener('fetch', event => {});`
 				{ filePath: "boop/file-3.txt", content: "Content of file-3" },
 				{ filePath: "file-4.txt", content: "Content of file-4" },
 				{ filePath: "beep/file-5.txt", content: "Content of file-5" },
-				{ filePath: "beep/file-6.txt", content: "Content of file-1" },
+				{
+					filePath: "beep/boop/beep/boop/file-6.txt",
```

Comment:
Is the idea here to have a file whose path is longer than 16 bytes?

Reply:
Not specifically, I just through I should probably chuck in a more nested test case in there